pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -128,7 +128,7 @@ func RunNewBuild(fullName string, f *clientcmd.Factory, out io.Writer, in io.Rea
 	for _, item := range result.List.Items {
 		switch t := item.(type) {
 		case *buildapi.BuildConfig:
-			fmt.Fprintf(c.Out(), "A build configuration was created - you can run `%s start-build %s` to start it.\n", fullName, t.Name)
+			fmt.Fprintf(c.Out(), "A build configuration was created and an initial build triggered - you can run `%s build-logs %s-1` to stream the logs as the build progresses.\n", fullName, t.Name)
```

Comment:
Are we deleting old builds yet? Are you sure the build suffix will be always -1 ? If I do new-build and then I delete my BuildConfig and do new-build again, will the name of the build have -1 suffix?